features/all-positions/AllPositions.tsx

```diff
@@ -46,20 +53,28 @@ const AllPositions = () => {
     return utils.commify(x as string);
   };
 
+  const prettyAddress = (x: String | null) => {
+    if (!x) return "N/A";
+    return x.substr(0, 6) + "..." + x.substr(x.length - 6, x.length);
+  };
+
   const getCollateralRatio = (
     collateral: BigNumberish,
     tokens: BigNumberish
   ) => {
     if (!latestPrice) return null;
     const tokensScaled = Number(tokens) * Number(latestPrice);
-    return Number(collateral) / tokensScaled;
+    return (Number(collateral) / tokensScaled).toFixed(4);
   };
 
   return (
     <Container>
       <Box>
         <Typography>
-          <i>Estimated prices are sourced from the latest trade data on: </i>
+          <i>
+            Estimated price of {latestPrice ? latestPrice : "N/A"} for{" "}
```

Comment:
Can you also round `latestPrice` to 4 decimal places?

Reply:
sure & done :)